can_tools/scrapers/official/TN/tn_state.py

```diff
@@ -0,0 +1,208 @@
+import pandas as pd
+import requests
+import us
+
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import CountyDashboard
+from can_tools.scrapers.official.base import StateDashboard
+
+
+class TennesseeCasesDeathsTests(CMU, StateDashboard):
+    """
+    Fetch state level covid data from official state of Tennessee spreadsheet
+    """
+
+    source = (
+        "https://www.tn.gov/content/tn/health/cedep/ncov/data.html"
+        "https://www.tn.gov/health/cedep/ncov/data/downloadable-datasets.html"
+    )
+    state_fips = int(us.states.lookup("Tennessee").fips)
+    has_location = True
+    location_type = "state"
+
+    def fetch(self) -> requests.Response:
+        # Set url of downloadable dataset
+        url = (
+            "https://www.tn.gov/content/dam/tn/health/documents/cedep/"
+            "novel-coronavirus/datasets/Public-Dataset-Daily-Case-Info.XLSX"
+        )
+        request = requests.get(url)
+        return request.content
+
+    def normalize(self, data) -> pd.DataFrame:
+        # Read data into data frame
+        df = pd.read_excel(data, parse_dates=["DATE"])
+
+        # Rename columns
+        df = df.rename(columns={"DATE": "dt"})
+
+        # Create dictionary for columns to map
+        crename = {
+            "TOTAL_CONFIRMED": CMU(
+                category="cases", measurement="cumulative", unit="people"
+            ),
+            "TOTAL_PROBABLE": CMU(category="cases", measurement="new", unit="people"),
+            "TOTAL_DEATHS": CMU(
+                category="deaths", measurement="cumulative", unit="people"
+            ),
+            "NEW_DEATHS": CMU(category="deaths", measurement="new", unit="people"),
+            "POS_TESTS": CMU(
```

Comment:
Is there any documentation from TN that can help us figure out what type of test it is?

Reply:
Thanks for asking me to investigate a bit more.  I've found these answers, I think we can safely call this a PCR test.  https://www.tn.gov/content/dam/tn/health/documents/cedep/novel-coronavirus/Regional_Rate_Map_FAQ.pdf
https://www.tn.gov/health/cedep/ncov/data/data-faqs.html (under What is a confirmed case?)  I'll update accordingly.